packages/expo-file-system/src/next/ExpoFileSystem.types.ts

```diff
@@ -225,3 +232,42 @@ export declare class FileHandle {
    */
   size: number | null;
 }
+
+export type FileInfo =
+  /**
+   * Object returned when file exist.
+   */
+  | {
```

Comment:
We don't need two separate types for `FileInfo`. We can keep one type and have exists as a `boolean` this way it will render nicely in the docs

```suggestion
/**
 * Object containing information about the file.
*/
export type FileInfo = {
```